docs/onboarding/8_16/operator/onprem_kube_stack_values.yaml

```diff
@@ -0,0 +1,460 @@
+              from_attribute: host.id
+              action: insert
+        resource/process:
+          attributes:
+            - key: process.executable.name
+              action: delete
+            - key: process.executable.path
+              action: delete
+        resourcedetection/system:
+          detectors: ["system", "ec2"]
+          system:
+            hostname_sources: [ "os" ]
+            resource_attributes:
+              host.name:
+                enabled: true
+              host.id:
+                enabled: false
+              host.arch:
+                enabled: true
+              host.ip:
+                enabled: true
+              host.mac:
+                enabled: true
+              host.cpu.vendor.id:
+                enabled: true
+              host.cpu.family:
+                enabled: true
+              host.cpu.model.id:
+                enabled: true
+              host.cpu.model.name:
+                enabled: true
+              host.cpu.stepping:
+                enabled: true
+              host.cpu.cache.l2.size:
+                enabled: true
+              os.description:
+                enabled: true
+              os.type:
+                enabled: true
+          ec2:
+            resource_attributes:
+              host.name:
+                enabled: false
+              host.id:
+                enabled: true
+        k8sattributes:
+          filter:
+            node_from_env_var: K8S_NODE_NAME
+          passthrough: false
+          pod_association:
+            - sources:
+                - from: resource_attribute
+                  name: k8s.pod.ip
+            - sources:
+                - from: resource_attribute
+                  name: k8s.pod.uid
+            - sources:
+                - from: connection
+          extract:
+            metadata:
+              - "k8s.namespace.name"
+              - "k8s.deployment.name"
+              - "k8s.replicaset.name"
+              - "k8s.statefulset.name"
+              - "k8s.daemonset.name"
+              - "k8s.cronjob.name"
+              - "k8s.job.name"
+              - "k8s.node.name"
+              - "k8s.pod.name"
+              - "k8s.pod.ip"
+              - "k8s.pod.uid"
+              - "k8s.pod.start_time"
+            labels:
+              - tag_name: app.label.component
+                key: app.kubernetes.io/component
+                from: pod
+      receivers:
+        otlp:
+          protocols:
+            grpc:
+              endpoint: 0.0.0.0:4317
+            http:
+              endpoint: 0.0.0.0:4318
+        filelog:
+          retry_on_failure:
+            enabled: true
+          start_at: end
+          exclude:
+          # exlude collector logs
+          - /var/log/pods/opentelemetry-operator-system_opentelemetry-kube-stack*/*/*.log
+          include:
+          - /var/log/pods/*/*/*.log
+          include_file_name: false
+          include_file_path: true
```

Comment:
is there any reason to not use `storage: file_storage` and corresponding extention? (it was used in the elastic-otel-collector manifest)

Reply:
it was added by setting `storeCheckpoints: true` in presets
```
presets:
      logsCollection:
        enabled: true
        storeCheckpoints: true
```

Reply:
:+1:  
  Is there any rotation policy for the generated files? Maybe @ChrsMark has more context on why we included in the initial k8s manifests

Reply:
We definitely need this in order to ensure that we properly keep track of log file offsets upon Collector's restarts.

Reply:
Makes sense to me, thank you both! @tetianakravchenko can we resolve the conversation? `storeCheckPoints` is now enabled in the configuration